### PR TITLE
Fix cos image project to cos-cloud.

### PR DESCRIPTION
**Pull request by Random-Liu**
Addressed https://github.com/kubernetes/kubernetes/pull/45136#discussion_r118092211.

@vishh @yujuhong @dchen1107

### CODE REVIEW

File: hack/make-rules/test-e2e-node.sh [hunk 1]
```diff
@@ -73,9 +73,9 @@ if [ $remote = true ] ; then
   gubernator=${GUBERNATOR:-"false"}
   image_config_file=${IMAGE_CONFIG_FILE:-""}
   if [[ $hosts == "" && $images == "" && $image_config_file == "" ]]; then
-    image_project=${IMAGE_PROJECT:-"google-containers"}
+    image_project=${IMAGE_PROJECT:-"cos-cloud"}
     gci_image=$(gcloud compute images list --project $image_project \
-    --no-standard-images --regexp="gci-dev.*" --format="table[no-heading](name)")
+    --no-standard-images --regexp="cos-dev.*" --format="table[no-heading](name)")
```
Comment (yujuhong) [inline hack/make-rules/test-e2e-node.sh]: The benchmark-config uses cos-beta-* images though.

Comment (Random-Liu) [inline hack/make-rules/test-e2e-node.sh]: I don't know why we use dev, that's why I'm looking for vishh and want to ask him why we should use dev here.
https://github.com/Random-Liu/kubernetes/commit/e9034f1e0aeace5b7a0847101a19397f201257d6#diff-d06d69f47fb79d27f9f55ad0d01be769

Comment (vishh) [inline hack/make-rules/test-e2e-node.sh]: `dev` was a thing of the past. The idea was to test `dev` images as soon as they are released. Now, we can switch to `beta` I hope. We aren't updating gci images as frequently as we used to do.

**Comment by vishh** [comment]
Thanks for this PR lantao.

File: hack/make-rules/test-e2e-node.sh (hunk 1 above)
Comment (Random-Liu) [inline hack/make-rules/test-e2e-node.sh]: @vishh ACK. Then will change to beta. :)

Comment (Random-Liu) [inline hack/make-rules/test-e2e-node.sh]: Done.

**Comment by Random-Liu** [comment]
@vishh Np~

**Comment by Random-Liu** [comment]
@fejta Ask for approval. :)

**Pull request merged**